bookwyrm/tests/importers/test_goodreads_import.py

```diff
@@ -47,7 +47,17 @@ def setUp(self):
 
     def test_create_job(self):
         """creates the import job entry and checks csv"""
-        import_job = self.importer.create_job(self.user, self.csv, False, "public")
+        job_options = {
+            "include_reviews": False,
+            "privacy": "public",
+        }
+        import_options = {
+            "file_type": "csv",
+            "default_shelf": None,
+        }
+        import_job = self.importer.create_job(
+            self.user, self.csv, job_options=job_options, import_options=import_options
+        )
```

Comment:
As can be seen here, I grouped some arguments in named arguments in order to avoid the following pylint error

```
bookwyrm/importers/importer.py:20:4: R0913: Too many arguments (7/5) (too-many-arguments)
```

I don't know if it's the best (or most pythonic?) way to do it. It's the most practical way I found to pass the default shelf and the file type.

Reply:
I don't know if there's a definitive right answer here, but I think it's clearer to use un-grouped arguments rather than dictionaries, and just disable the warning (add `# pylint: disable=too-many-arguments` above the line it's complaining about). That way it's clearer which arguments are required and easier to see if anything that should be passed in is missing.

Reply:
Will do, thanks!